docs/en/relying-party-solution.rst

```diff
@@ -98,15 +98,15 @@ The payload of the QR Code is a **Base64 encoded string** based on the following
 
   eudiw://authorize?client_id=`$client_id`&request_uri=`$request_uri`
 
-In the Same Device Flow the parameter **client_id** and **request_uri** are the same if the ones used in the Cross Device Flow with the only difference about the url schema and the removal of the Verifier's FQDN from the URL. 
+In the Same Device Flow the parameter **client_id** and **request_uri** are the same if the ones used in the Cross Device Flow with the only difference about the url schema and the removal of the Verifier's FQDN from the URL.
 
 In the Same Device Flow the Relying Party uses a HTTP response redirect (status code 302) to give to the Wallet Instance the resource where the request object is available for download, as represented in the following non-normative example:
 
 
 .. code:: text
 
     HTTP/1.1 /pre-authz-endpoint Found
-    Location: https://verifier.example.org/request_uri_endpoint?
+    Location: eudiw://authorize?
```

Comment:
Please leave https:// which is a Universal link for the app. You cannot use a deep link. We'll make a proposal on that but I wouldn't change the schema for now

Reply:
ok, so a change is then required since we cannot keep verifier.example.org, isn't that true?

do you have any proposal for this universal link, like `https://eudi.wallet.gov.it`?

Reply:
Correct, our idea was to have a universal link (managed by trust anchors) that brings together all the wallet providers (an RFC on this will follow). So for now an idea could be` https://wallet-providers.eudi.wallet.gov.it`

Reply:
Why don't we just leave it unchanged for now? So we can merge this PR.

We can then propose changes in a new PR

@peppelinux